src/main/java/org/jabref/gui/util/ThemeLoader.java

```diff
@@ -37,12 +38,18 @@
 public class ThemeLoader {
 
     private static final String DEFAULT_PATH_MAIN_CSS = JabRefFrame.class.getResource("Base.css").toExternalForm();
-    private static final String CSS_SYSTEM_PROPERTY = System.getProperty("jabref.theme.css");
     private static final Logger LOGGER = LoggerFactory.getLogger(ThemeLoader.class);
+    private final String CSS_SYSTEM_PROPERTY;
     private final FileUpdateMonitor fileUpdateMonitor;
 
-    public ThemeLoader(FileUpdateMonitor fileUpdateMonitor) {
+    public ThemeLoader(FileUpdateMonitor fileUpdateMonitor, JabRefPreferences jabRefPreferences) {
         this.fileUpdateMonitor = Objects.requireNonNull(fileUpdateMonitor);
+
+        String cssFileName = jabRefPreferences.get(JabRefPreferences.FX_THEME);
+        if (cssFileName != null) {
+            CSS_SYSTEM_PROPERTY = new File("src/main/java/org/jabref/gui/" + cssFileName).getAbsolutePath();
```

Comment:
I think we misunderstood us ;) `Paths.get(JabRefFrame.class.getResource(cssFileName).toURI()) `
 This is what I wanted to point to